base/src/main/java/org/geysermc/api/util/ApiVersion.java

```diff
@@ -0,0 +1,46 @@
+package org.geysermc.api.util;
+
+public class ApiVersion {
+    private final int major;
+    private final int minor;
+    private final int patch;
+
+    public ApiVersion(int major, int minor, int patch) {
+        this.major = major;
+        this.minor = minor;
+        this.patch = patch;
+    }
+
+    /**
+     * Returns the major version of the api.
+     * @return the major version
+     */
```

Comment:
Space between the method description and the tags, similarly with the other methods